crates/xtask-bump-check/src/xtask.rs

```diff
@@ -158,15 +158,16 @@ fn bump_check(args: &clap::ArgMatches, config: &cargo::util::Config) -> CargoRes
     config.shell().status("Running", &cmd)?;
     cmd.exec()?;
 
-    if let Some(referenced_commit) = referenced_commit.as_ref() {
-        let mut cmd = ProcessBuilder::new("cargo");
-        cmd.arg("semver-checks")
-            .arg("--workspace")
-            .arg("--baseline-rev")
-            .arg(referenced_commit.id().to_string());
-        config.shell().status("Running", &cmd)?;
-        cmd.exec()?;
-    }
+    // This test does not work because we are running it against a too old version of cargo.
+    // if let Some(referenced_commit) = referenced_commit.as_ref() {
+    //     let mut cmd = ProcessBuilder::new("cargo");
+    //     cmd.arg("semver-checks")
+    //         .arg("--workspace")
+    //         .arg("--baseline-rev")
+    //         .arg(referenced_commit.id().to_string());
+    //     config.shell().status("Running", &cmd)?;
+    //     cmd.exec()?;
+    // }
 
```

Comment:
Can this be made less 'intrusive' using a `#[cfg()]` annotation? Maybe something like
```
#[cfg(target_os = "solana")]
```
before the `if-`statement? Since such a `target_os` doesn't exist for host compilation, the following `if-`statement will always be excluded from compiled code.